docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - ~/volumes/ci/roger:/roger/
```

Comment:
~/ might work then, just place the yaml there. Still need to handle that file manually, so making a dir might not be that bad